alacritty_terminal/src/display.rs

```diff
@@ -224,6 +224,44 @@ impl Display {
             api.clear(background_color);
         });
 
+        // We should call `clear` when window is offscreen, so when `window.show()` happens it
+        // would be already with background color instead of bunch of borders + uninitialized
```

Comment:
`bunch of borders` is implementation specific to the WM which is used, if your WM has no borders around your windows, Alacritty will not just render the borders..

So just calling it an `unitialized surface` should be enough, without any mention of borders.